### ISSUE TITLE
[Feature] Allow relative path to XSL file

### ISSUE THREAD

**Issue by boazpoolman** [ekalinin/sitemap.js#367]
The `xslUrl` option can only be a full URL.
I would like to add the XSL relative to the sitemap file. Like so:
```
const sitemap = new SitemapStream({
  hostname: 'www.github.com',
  xslUrl: "/sitemap.xsl",
});
```

Right now this value is validated with the `url` package like so:
```
// Throws if url is invalid
new URL(url);
```

I would suggest to remove this validation and make the xslUrl option a free text field.

Looking forward to your response! :)

**Comment by boazpoolman** [comment]
Fix in #368

**Comment by derduher** [comment]
looks like the fix was merged
